### PR TITLE
Move fingerprinting services from custom package to main package

### PR DESCRIPTION
**Pull request by lptr**
This solves class cycle issues, and output snapshotting used to live in the main package anyway, so it's more consistent, too.

### CODE REVIEW

**Comment by lptr** [comment]
@bot-gradle test QFL please.

File: subprojects/execution/src/main/java/org/gradle/internal/execution/InputFingerprinter.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.gradle.internal.execution.fingerprint;
+package org.gradle.internal.execution;
```
[the remark this answers is not in the document]
Comment (lptr) [reply subprojects/execution/src/main/java/org/gradle/internal/execution/InputFingerprinter.java:17]: I think it's not too crowded in the main package even after everything's moved up. I am planning some more major refactors later on, so I wouldn't stress too much on the actual package structure right now. Once the architecture becomes stable (more stable), I think it'll be time to move some inner classes to top-level. That might be a good time to think about splitting things into meaningful packages.

**Comment by lptr** [comment]
@bot-gradle test and merge please.

**Comment by wolfs** [comment]
@bot-gradle test and merge

**Pull request merged**